app/gateway/cp-dp-communication.md

```diff
@@ -13,28 +13,427 @@ works_on:
+         19:8f:64:ee:82:a6:e8:e6:de:57:a7:24:82:72:82:49
+```
+{% endnavtab %}
+
+{% navtab "Certificate on DP" %}
+Here is an example of a certificate on a Data Plane:
+
+```
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            4d:8b:eb:89:a2:ed:b5:29:80:94:31:e4:94:86:ce:4f:98:5a:ad:a0
+        Signature Algorithm: ecdsa-with-SHA256
+        Issuer: CN = Hybrid Intermediate CA
+        Validity
+            Not Before: Jul 31 00:57:01 2020 GMT
+            Not After : Oct 29 00:57:31 2020 GMT
+        Subject: CN = kong-dp-ce39edecp.service
+        Subject Public Key Info:
+            Public Key Algorithm: id-ecPublicKey
+                Public-Key: (256 bit)
+                pub:
+                    04:19:51:80:4c:6d:8c:a8:05:63:42:71:a2:9a:23:
+                    34:34:92:c6:2a:d3:e5:15:6e:36:44:85:64:0a:4c:
+                    12:16:82:3f:b7:4c:e1:a1:5a:49:5d:4c:5e:af:3c:
+                    c1:37:e7:91:e2:b5:52:41:a0:51:ac:13:7b:cc:69:
+                    93:82:9b:2f:e2
+                ASN1 OID: prime256v1
+                NIST CURVE: P-256
+        X509v3 extensions:
+            X509v3 Key Usage: critical
+                Digital Signature, Key Encipherment, Key Agreement
+            X509v3 Extended Key Usage:
+                TLS Web Client Authentication
+            X509v3 Subject Key Identifier:
+                25:82:8C:93:85:35:C3:D6:34:CF:CB:7B:D6:14:97:46:84:B9:2B:87
+            X509v3 Authority Key Identifier:
+                keyid:16:0D:CF:92:3B:31:B0:61:E5:AB:EE:91:42:B9:60:56:0A:88:92:82
+            X509v3 CRL Distribution Points:
+
+                Full Name:
+                  URI:https://crl-service.yourcorp.tld/v1/pki/crl
+
+    Signature Algorithm: ecdsa-with-SHA256
+         30:44:02:20:65:2f:5e:30:f7:a4:28:14:88:53:58:c5:85:24:
+         35:50:25:c9:fe:db:2f:72:9f:ad:7d:a0:67:67:36:32:2b:d2:
+         02:20:2a:27:7d:eb:75:a6:ee:65:8b:f1:66:a4:99:32:56:7c:
+         ad:ca:3a:d5:50:8f:cf:aa:6d:c2:1c:af:a4:ca:75:e8
+```
+{% endnavtab %}
+{% endnavtabs %}
+
+#### (Optional) Revocation checks of Data Plane certificates
+
+When {{site.base_gateway}} is running hybrid mode with PKI mode, the Control Plane can be configured to
+optionally check for revocation status of the connecting Data Plane certificate.
+
+The supported method is through Online Certificate Status Protocol (OCSP) responders.
+Issued Data Plane certificates must contain the Certificate Authority Information Access extension
+that references the URI of OCSP responder that can be reached from the Control Plane. [`cluster_ocsp`](/gateway/configuration/#cluster-ocsp) affects all hybrid mode connections established from a Data Plane to its Control Plane.
+
+To enable OCSP checks, set the `cluster_ocsp` config on the Control Plane to one of the following values:
+
+<!--vale off-->
+{% kong_config_table %}
+config:
+  - name: cluster_ocsp
+{% endkong_config_table %}
+<!--vale on-->
```

Comment:
```suggestion
```

Reply:
I removed this because it felt a bit too how-to to fit into this doc. If you agree, make a ticket for this stuff for post ga.